lib/api.common.js

```diff
@@ -594,18 +594,17 @@ api.common.sortCompareByName = (
   return 0;
 };
 
-api.common.clearCacheStartingWith = (
+api.common.clearCacheStartsWith = (
```

Comment:
I removed this function at all, but Maps coming from factory `api.common.cache()` have `add`, `del` and `clr` methods